test_scripts/Resumption/InteriorVehicleData/018_InteriorVD_resumption_two_apps_same_moduleTypes_different_moduleIds_IGN_OFF.lua

```diff
@@ -0,0 +1,72 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description: Successful resuming of interior vehicle data for two apps after IGN_OFF in case apps are subscribed to
+--  the same moduleTypes with different moduleIds
+--
+-- Precondition:
+-- 1. HMI and SDL are started
+-- 2. Mobile app1 and app2 with REMOTE_CONTROL hmi type are registered and activated
+-- 3. App1 is subscribed to module_1 with moduleId_1
```

Comment:
please change all module_1 to moduleType_1

Reply:
Done